test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1829,3 +1842,93 @@ UCS_TEST_SKIP_COND_P(test_ucp_address_v2, diff_seg_sizes,
 }
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_address_v2)
+
+class test_pci_bw : public test_ucp_wireup {
+public:
+    static void get_test_variants(std::vector<ucp_test_variant> &variants)
```

Comment:
Please apply `const` and/or `auto` wherever possible.

Reply:
this function is inherited from a base class:
```
class test_ucp_context : public ucp_test {
public:
    static void get_test_variants(std::vector<ucp_test_variant> &variants);
}
```